src/SetupRemote/Product.wxs

```diff
@@ -74,6 +78,7 @@
         </ServiceInstall>
         <ServiceControl Id="RemoteRBrokerServiceUninstall" Name="RHostBrokerService" Remove="uninstall" Stop="uninstall"/>
         <fire:FirewallException Id="RBrokerSvcFirewallException" Name="Microsoft.R.Host.Broker" Program="[INSTALLFOLDER]Microsoft.R.Host.Broker.exe" Scope="any" Profile="all"/>
+        <util:EventSource Log="Application" Name="RHostBrokerService" EventMessageFile="[NETFRAMEWORK40CLIENTINSTALLROOTDIR]EventLogMessages.dll"/>
```

Comment:
.net 4.6 is required before installing. The check for this is in the bundle project.